agenthub/micro/coder/prompt.md

```diff
@@ -21,7 +21,7 @@ Do NOT finish until you have completed the tasks.
 
 ## History
 {{ instructions.history_truncated }}
-{{ history_to_json(state.history[-10:]) }}
+{{ history_to_json(state.history, max_events=20) }}
```

Comment:
Why are you passing the default value of max_events again?

Reply:
I think it makes it explicit to people who want to write new microagents that this is configurable. I would personally vote for keeping it.